cmd/metrics-scraper/app/scrape/other.go

```diff
@@ -0,0 +1,258 @@
+package scrape
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	_ "github.com/glebarez/sqlite"
+	v1 "github.com/karmada-io/dashboard/cmd/api/app/types/api/v1"
```

Comment:
we should not reused the router in api module, here we need another copy of router